### PR TITLE
Fix layout and visual regressions around default avatars

### PR DESCRIPTION
**Pull request by t3chguy**
Regressed by https://github.com/matrix-org/matrix-react-sdk/commit/a8aa4de4b4c14a6a995071aab3f2f871b3ff2388
Fixes https://github.com/vector-im/element-web/issues/24375
Fixes https://github.com/vector-im/element-web/issues/24369

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix layout and visual regressions around default avatars ([\#10031](https://github.com/matrix-org/matrix-react-sdk/pull/10031)). Fixes vector-im/element-web#24375 and vector-im/element-web#24369.

### CODE REVIEW

**Review by richvdh — COMMENTED** [review]
The TSX changes look good.

The PCSS changes I could do with some more explanation on: why are they needed? Could they have some comments explaining what they are for?

File: res/css/structures/_SpacePanel.pcss [hunk 1]
```diff
@@ -277,14 +277,11 @@ $activeBorderColor: $primary-content;
...
             border-radius: 8px;
-            background-color: $panel-actions;
-            font-size: $font-15px !important; /* override inline style */
             font-weight: $font-semi-bold;
             line-height: $font-18px;
-
-            & + .mx_BaseAvatar_image {
-                visibility: hidden;
-            }
+            /* override inline styles which are part of the default avatar style as these uses a monochrome style */
```
Comment (richvdh) [inline res/css/structures/_SpacePanel.pcss:282]: it feels like we shouldn't be using inline styles, *especially* if they they have to be overridden in the css file. but I guess that's an existing situation which isn't getting materially worse here.

**Review by richvdh — APPROVED** [review]
I still have no real idea why these CSS changes are now necessary, but it seems plausible I guess.

File: res/css/structures/_SpacePanel.pcss (hunk 1 above)
Comment (t3chguy) [reply res/css/structures/_SpacePanel.pcss:282]: We can't avoid using inline styles for things where the colours/sizes are defined in Javascript. The space panel is the one place which doesn't use coloured default avatars, we could override them in JS but this seems cleaner.

Comment (richvdh) [reply res/css/structures/_SpacePanel.pcss:282]: Hrm, I guess. Does the font size also need to be defined in javascript?

(Also, I wonder if we should have a comment at the point that the styles are set in javascript that they will be overridden for the space panel)

Comment (t3chguy) [reply res/css/structures/_SpacePanel.pcss:282]: not sure about the font size, that one is quite old

**Pull request merged**